internal/backends/winit/winitwindowadapter.rs

```diff
@@ -332,6 +332,11 @@ impl<Renderer: WinitCompatibleRenderer + 'static> WindowAdapterSealed
         winit_window.set_title(&window_item.title());
         winit_window
             .set_decorations(!window_item.no_frame() || winit_window.fullscreen().is_some());
+        winit_window.set_window_level(if window_item.always_on_top() {
+            winit::window::WindowLevel::AlwaysOnTop
+        } else {
+            winit::window::WindowLevel::Normal
+        });
```

Comment:
Question: Does being fullscreen matter like it does for `no-frame`?

Reply:
I don't think it matters :)